clinvar_ingest/reader.py

```diff
@@ -92,6 +97,7 @@ def read_clinvar_xml(reader: TextIO, disassemble=True) -> Iterator[model.Model]:
         else:
             raise ValueError(f"Unexpected event: {event}. Element: {ET.tostring(elem)}")
 
+        # I'd also make these "tags" enums of some kind.
```

Comment:
This is a good idea for the XML tags. There are a lot of XML elements we look at though, using string literals, and I probably wouldn't model all of them as constants.

For the `iterparse` events, the strings are always just "start" and "end", I'm good with the idea to use StrEnum there